paperqa/types.py

```diff
@@ -40,6 +40,10 @@
 logger = logging.getLogger(__name__)
 
 
+ENV_VAR_MATCH: Collection[str] = {"1", "true"}
+ENV_VAR_MISMATCH: Collection[str] = {"0", "false"}
```

Comment:
So we are using these now for both environment variables and manifest fields.

Do you mind renaming these to `VAR_MATCH_LOOKUP` and `VAR_MISMATCH_LOOKUP`, mainly since they're more general than just environment variables